### PR TITLE
Spanish translation for "Preparing for a Tech Talk, Part 2: What, Why, and How"

### PR DESCRIPTION
**Pull request by Naahuel**
This was a tricky one.

- I've translated the images too. If you're not pleased with the results let me know (I keep the editable template).
- Note the filenames for translated images, hopefully they're ok.
- I added some minor corrections to part 1.
- I made sure that links between the series (part 1 and part 2) are kept in Spanish.

Thanks!

### CODE REVIEW

**Comment by Naahuel** [comment]
Looks good? Locally it doesn't work that well, it adds the page slug twice:
screenshot from 2019-01-18 17-44-12

Am I missing something?

**Comment by gaearon** [comment]
Yea master looks hosed. Will fix.

File: src/pages/preparing-for-tech-talk-part-1-motivation/preparing-for-tech-talk-part-1-motivation.es.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ langs: ['en', 'es']
...
-He hecho unas [cuantas](https://www.youtube.com/watch?v=xsSnOQynTHs) [charlas](https://www.youtube.com/watch?v=nLF0n9SACd4) [técnicas](https://www.youtube.com/watch?v=dpw9EHDh2bM) que creo que fueron bien.
+He dado unas [cuantas](https://www.youtube.com/watch?v=xsSnOQynTHs) [charlas](https://www.youtube.com/watch?v=nLF0n9SACd4) [técnicas](https://www.youtube.com/watch?v=dpw9EHDh2bM) que creo que fueron bien.
 
 A veces la gente me pregunta cómo me preparo para una charla. Para cada orador, la respuesta es muy personal. Solo voy a compartir lo que funciona para mí.
 
 **Este es el primer artículo de una serie** donde explico mi proceso de preparación para una carla técnica — desde la concepción de la idea hasta el día de la presentación:
 
 * **Preparándose para una charla técnica, Parte 1: Motivación (*este artículo*)**
-* **[Preparándose para una charla técnica, Parte 2: Qué, Por Qué y Cómo](/preparing-for-tech-talk-part-2-what-why-and-how/)**
+* **[Preparándose para una charla técnica, Parte 2: Qué, Porqué y Cómo](/es/preparing-for-tech-talk-part-2-what-why-and-how/)**
```
Comment (davidtaboas) [inline src/pages/preparing-for-tech-talk-part-1-motivation/preparing-for-tech-talk-part-1-motivation.es.md]: Debería ser "Qué, por qué y cómo"

Comment (davidtaboas) [reply src/pages/preparing-for-tech-talk-part-1-motivation/preparing-for-tech-talk-part-1-motivation.es.md]: Si lo ves consistente así, estaría bien. Sino yo tiraría por "Qué, por qué y cómo" porque son preguntas indirectas. Luego en el texto si se utiliza como "razón" pues utilizaría porqué si es la única posibilidad.

**Comment by Naahuel** [comment]
Added a small change in the titles of part 1 and 2 (based on @davidtaboas 's suggestions) and renamed to new file structure.

File: src/pages/preparing-for-tech-talk-part-1-motivation/index.es.md [hunk 2]
```diff
@@ -78,8 +78,8 @@ Así que si quieres dar una charla, *hablar* con gente es una buena forma de emp
...
+En este artículo he descrito el marco de referencia que me resulta útil para pensar en ideas para charlas. De nuevo, quiero enfatizar que solo estoy compartiendo lo que funciona para mi, hay muchos [tipos de charlas](https://mobile.twitter.com/jackiehluo/status/1077717283026411520) y tu mirada en esto puede ser muy diferente.
 
-En el siguiente artículo de esta serie, hablaré sobre la preparación del bosquejo de la charla, las diapositivas, el ensayo de las charlas y lo que hago el día de la presentación.
+En los siguientes artículos de esta serie, hablaré sobre la preparación del bosquejo de la charla, las diapositivas, el ensayo de las charlas y lo que hago el día de la presentación.
 
-**A continuación en esta serie: [Preparándose para una charla técnica, Parte 2: Qué, Por Qué y Cómo](/preparing-for-tech-talk-part-2-what-why-and-how/)**
+**A continuación en esta serie: [Preparándose para una charla técnica, Parte 2: Qué, Por Qué y Cómo](/es/preparing-for-tech-talk-part-2-what-why-and-how/)**
```
Comment (gaearon) [inline src/pages/preparing-for-tech-talk-part-1-motivation/index.es.md]: I'd like to keep links pointing to English URLs. We can later make this automatic.

**Comment by gaearon** [comment]
Thanks

**Pull request merged**